frontend/src/views/camp/CampProgram.vue

```diff
@@ -103,5 +115,11 @@ export default {
       }
     },
   },
+  methods: {
+    showUnlockReminder(move) {
+      this.reminderType = move ? 'move' : 'create'
```

Comment:
How about just storing `reminderText` instead of introducing an internal enum-like state?
```suggestion
      this.reminderText = move ? $tc('...') : $tc('...')
```
The downside is that the translation won't change when switching the locale. But in production, the locale can't be switched on the picasso view.